037-WindowsVirtualDesktop/Student/00-Pre-Reqs.md

```diff
@@ -0,0 +1,34 @@
+# Challenge 0: Getting started
```

Comment:
This needs to match the table of contents.... same applies for all challenges so I'll only make this comment once.